src/org/omegat/core/data/PluginInformation.java

```diff
@@ -42,56 +53,107 @@
     private static final String BUNDLE_NAME = "Bundle-Name";
     private static final String BUILT_BY = "Built-By";
 
+    public enum STATUS {
+        INSTALLED,
+        BUNDLED,
+        UPGRADABLE,
+        UNINSTALLED
```

Comment:
`UPGRADABLE` and `UNINSTALLED` are not used (sorry I haven't seen if they're used in your other related PR).

I can imagine how `UPGRADABLE` will be used. But what about `UNINSTALLED`? How does that work?

Reply:
Yes, `UNINSTALLED` is not used. I'd like to remove it.

Reply:
Now we don't have plugin removal feature on Preference - Plugins.
When we got the feature, preference dialog may show "UNINSTALLED" status before restart OmegaT.